README.md

```diff
@@ -72,7 +73,6 @@ releasify publish [--path|-p <path>]             ➡ The path to the project to
                   [--gh-group-by-label|-l <label>] ➡ Group the commits in the changelog message by PR's labels
                   [--npm-access|-a <string>]       ➡ It will set the --access flag of `npm publish` command. Useful for scoped modules. The value must be [public, restricted]
                   [--npm-dist-tag <string>]        ➡ It will add a npm tag to the module, like `beta` or `next`
-                  [--npm-otp <code>]               ➡ It will provide the otp code to the npm publish
```

Comment:
please keep this property

Reply:
oh yeah I wanted to comment here just before the standup started that we are probably using this on some CIs, will retain this